### ISSUE TITLE
[BUG]: Issue with software rendering on Grand Theft Auto: San Andreas (SLUS-20946)

### ISSUE THREAD

**Issue by GitHubUser6721** [PCSX2/pcsx2#6920]
### Describe the Bug

There is this pretty annoying issue with the software rendering where the bottom right corner of the screen has this weird corrupted line, this issue doesn't happen on the Vulkan renderer or OpenGLrenderer and I've looked into it on original hardware and I can't see the issue happening on there (However I was in a building in-game but I do remember a while back I was in the same building on PCSX2 and the issue was still there and I was using a CRT monitor for the PS2)

Honestly if somebody could double check that this issue occurs on original hardware with a more modern monitor that would also be good.

### Reproduction Steps

1. Load up Grand Theft Auto: San Andreas (SLUS-20946)
2. Switch to software rendering mode (Or switch before you load up the game)
3. Load into the game (Preferably start a new game) and you'll see the corruption in the bottom right corner

### Expected Behavior

Should appear how it appears on Vulkan or OpenGL

### PCSX2 Revision

v1.7.3247

### Operating System

Windows 10 (64bit)

### CPU

Intel(R) Core(TM) i7-2600 CPU @ 3.40GHz

### GPU

NVIDIA GeForce GTX 1660

### GS Settings

Display:
https://gyazo.com/c0049a65881cc9211ee2df165b8c6c45
Rendering:
https://gyazo.com/9ed56c877c6d2a36a61d6d11b14994e4
OSD:
https://gyazo.com/670c36bbbe09582fe70fa694c29b6d8e
Advanced:
https://gyazo.com/3c024c6756045aad5ecb139310335775

### Emulation Settings

Emulation:
https://gyazo.com/d68d365ed73fc1812135ca5fcc52ee1f
System:
https://gyazo.com/b4a85af7bc5549c4da3b3570ab84b76d

### GS Window Screenshots

Provided in "GS Settings"???

### Logs & Dumps

https://www.mediafire.com/file/a6kflmfwgbzcyqj/Grand_Theft_Auto_-_San_Andreas_SLUS-20946_20220826115512.gs/file

**Comment by refractionpcsx2** [comment]
Can you provide a screenshot of the error? I've never seen it before.

**Comment by GitHubUser6721** [comment]
It isn't an error it's a graphical issue

**Comment by refractionpcsx2** [comment]
A graphical issue is an error..  Anyway, semantics or not, can you show a screenshot of the error/problem/graphical error/leftover dinner, whatever you wanna call it.

**Comment by GitHubUser6721** [comment]
My bad lol it just clicked what you meant, sure here's a GIF that I think perfectly showcases the issue

https://www.mediafire.com/view/n8etlwsviwsii3l/SoftwareGlitch.gif/file

(Sorry for the download link, GIF was to big to upload because of the quality but if the quality was low you couldn't really see the issue)

**Comment by GitHubUser6721** [comment]
Uploaded it to Imgur because it wasn't loading through that link: https://imgur.com/a/ekIz7Pr

**Comment by refractionpcsx2** [comment]
yeah I had to download the gif to view it, I see it there.  Thanks
